gensim/test/test_direct_confirmation.py

```diff
@@ -23,7 +23,7 @@ def setUp(self):
 
     def testLogConditionalProbability(self):
         """Test log_conditional_probability()"""
-        obtained = direct_confirmation_measure.log_conditional_probability(self.segmentation, self.posting_list, self.num_docs)[0]
+        obtained = direct_confirmation_measure.log_conditional_probability(self.segmentation, self.posting_list, self.num_docs)[0][0]
```

Comment:
what is the reason to change the test? Please keep the test the same and avoid introducing the breaking change.